content/en/docs/tasks/security/authz-tcp/index.md

```diff
@@ -91,72 +91,65 @@ to the `v2` version, which talks to a MongoDB backend using TCP.
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: deny-all
+spec:
+  selector:
+    matchLabels:
+      app: mongodb
+EOF
 {{< /text >}}
 
+The command creates a `deny-all` policy that selects the MongoDB workload and will deny all requests because
+it doesn't have any rules.
+
 Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`).  You should see:
 
 * The **Book Details** section on the lower left of the page includes book type, number of pages, publisher, etc.
 * The **Book Reviews** section on the lower right of the page includes an error message **"Ratings service is
   currently unavailable"**.
 
-This is because Istio authorization is "deny by default", which means that you need to explicitly
-define access control policies to grant access to the MongoDB service.
-
 {{< tip >}}
 There may be some delays due to caching and other propagation overhead.
 {{< /tip >}}
 
-## Enforcing access control on TCP service
+## Enforcing access control on TCP workload
 
-Now let's set up service-level access control using Istio authorization to allow `v2` of `ratings`
-to access the MongoDB service.
+Now let's set up workload-level access control using Istio authorization to allow `v2` of `ratings`
+to access the MongoDB workload.
 
 Run the following command to apply the authorization policy:
 
 {{< text bash >}}
-$ kubectl apply -f @samples/bookinfo/platform/kube/rbac/mongodb-policy.yaml@
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: bookinfo-ratings-v2
+spec:
+  selector:
+    matchLabels:
+      app: mongodb
+  rules:
+  - from:
+    - source:
+        principals: ["cluster.local/ns/default/sa/bookinfo-ratings-v2"]
+    to:
+    - operation:
+        ports: ["27017"]
+EOF
 {{< /text >}}
 
-Once applied, the policy has the following effects:
-
-* Creates the following `mongodb-viewer` service role, which allows access to the MongoDB service on port 27017.
-
-    {{< text yaml >}}
-    apiVersion: "rbac.istio.io/v1alpha1"
-    kind: ServiceRole
-    metadata:
-      name: mongodb-viewer
-      namespace: default
-    spec:
-      rules:
-      - services: ["mongodb.default.svc.cluster.local"]
-        constraints:
-        - key: "destination.port"
-          values: ["27017"]
-    {{< /text >}}
-
-* Creates the following `bind-mongodb-viewer` service role binding, which assigns the `mongodb-viewer` role
-to the `bookinfo-ratings-v2` service.
-
-    {{< text yaml >}}
-    apiVersion: "rbac.istio.io/v1alpha1"
-    kind: ServiceRoleBinding
-    metadata:
-      name: bind-mongodb-viewer
-      namespace: default
-    spec:
-      subjects:
-      - user: "cluster.local/ns/default/sa/bookinfo-ratings-v2"
-      roleRef:
-        kind: ServiceRole
-        name: "mongodb-viewer"
-    {{< /text >}}
+The command creates a `bookinfo-ratings-v2` policy that selects the MongoDB workload and grants the access at port
```

Comment:
Please either include this detailed information before the command, merge it with the more general description above, or remove the detailed version here. I recommend adding the details here to the description above.